app/components/primer/alpha/action_list_item.html.erb

```diff
@@ -0,0 +1,59 @@
+<%= render(Primer::BaseComponent.new(tag: :li, **@system_arguments)) do %>
```

Comment:
This tag needs a `role` param.. I suppose with `listitem` as default but I may change that later :)

Reply:
Ok, I think it's better if the role is passed in, but we can use `listitem` as a default.